data/pathfinder/paizo/roleplaying_game/core_essentials/ce_abilities_race.lst

```diff
@@ -71,6 +71,10 @@ CATEGORY=Internal|Default.MOD															DEFINE:UseAutoReach|0
 CATEGORY=Internal|Default.MOD															DEFINE:PrimaryAttackDamageSize|0	DEFINE:PrimaryAttackDamageDice|0																																																ABILITY:Internal|AUTOMATIC|REACH_0|PREMULT:1,[PRESIZEEQ:F],[PRESIZEEQ:D],[PRESIZEEQ:T]|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_5|PREMULT:1,[PRESIZEEQ:S],[PRESIZEEQ:M]|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_10|PRESIZEEQ:L|PREVAREQ:RaceSizeIsLong,0|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_15|PRESIZEEQ:H|PREVAREQ:RaceSizeIsLong,0|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_20|PRESIZEEQ:G|PREVAREQ:RaceSizeIsLong,0|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_30|PRESIZEEQ:C|PREVAREQ:RaceSizeIsLong,0|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_5|PRESIZEEQ:L|PREVAREQ:RaceSizeIsLong,1|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_10|PRESIZEEQ:H|PREVAREQ:RaceSizeIsLong,1|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_15|PRESIZEEQ:G|PREVAREQ:RaceSizeIsLong,1|PREVAREQ:UseAutoReach,1	ABILITY:Internal|AUTOMATIC|REACH_20|PRESIZEEQ:C|PREVAREQ:RaceSizeIsLong,1|PREVAREQ:UseAutoReach,1
 CATEGORY=Internal|Default.MOD																																																																										ABILITY:Internal|AUTOMATIC|Size Tracker
 
+###Block: ABILITY:Internal|AUTOMATIC|Affinity Sorcerer Bonus|PREMULT:2,[PRECLASS:1,Sorcerer=1],[PREABILITY:1,CATEGORY=Special Ability,Sorcerer Bloodline ~ xxx]
```

Comment:
Sorcerer Bloodline ~ xxx? Did we miss something?

Reply:
Also, this won't work on a "BLOCK" line, it needs to be part of an existing ability or a MOD.

Reply:
This is a template for adding the Sorcerer Affinity Bonus to other races.